primitives/trie/src/key_nibbles.rs

```diff
@@ -1,6 +1,8 @@
 use std::borrow::Cow;
 use std::cmp;
 use std::fmt;
+
```

Comment:
Fixed in 0d3c6a30386845beb1387eb34f48fcfa76ecdeb3, this adds the unused import again

Reply:
Fixed.